src/picongpu/include/particles/ionization/None/None.def

```diff
@@ -27,9 +29,33 @@ namespace particles
 namespace ionization
 {
 
+    /** \struct None_Impl
```

Comment:
same here, better use a headline instead. brief is implicit when using a free line in between